src/jsonrpccpp/CMakeLists.txt

```diff
@@ -1,25 +1,12 @@
-find_package(JSONCPP REQUIRED)
-find_package(Threads REQUIRED)
-
 file(GLOB jsonrpc_source_common common/*.c*)
 file(GLOB jsonrpc_source_client client/*.c*)
 file(GLOB jsonrpc_source_server server/*.c*)
-list(APPEND jsonrpc_source_server server/abstractserver.h)
 
+# TODO not sure why some .h were not included
+# if we don't want to include it, remove it using list REMOVE_ITEM
 file(GLOB jsonrpc_header_common common/*.h)
```

Comment:
Files were not included beacuse they are not part of the public API. Which makes it more easy to change internal functionality. Please revert that change, or use remove for all the others.